### ISSUE TITLE
How to use DebugView?

### ISSUE THREAD

**Issue by laurinpaech** [tainicom/Aether.Physics2D#87]
Hey! We are currently using Aether as our physics engine and I'm trying to figure out how to make the DebugView work. Essentially we create a world with a bunch of bodies and pass our world to the Debugview and then we load the content and use RenderDebugData in our draw function:

`DebugView DebugView = new DebugView(World);
DebugView.LoadContent(gameInstance.GraphicsDevice, gameInstance.Content);`

Now DebugView complains that we don't have the DiagnosticsFont in our ContentManager. I added it to our Content but I dont think thats how its supposed to work?

We then try to draw the DebugView in our draw function with:
`DebugView.RenderDebugData(projection, view);`

It runs but we don't see any difference. Any help is appreciated!

(We use the tainicom.Aether.Physics2D.Diagnostics.MG NuGet package (along with the normal MG package) if that makes a difference.)

**Comment by SjaakAlvarez** [comment]
I use this code, in your case it might be different. Just remember that this is the last thing you should draw

``` C#
debug.AppendFlags(DebugViewFlags.Shape);
debug.AppendFlags(DebugViewFlags.CenterOfMass);
debug.AppendFlags(DebugViewFlags.DebugPanel);
debug.AppendFlags(DebugViewFlags.PerformanceGraph);


public override void Update(GameTime gameTime)
{
    debug.UpdatePerformanceGraph(_world.UpdateTime);
}

public void Draw(GameTime gameTime)
{
    Matrix i = Matrix.CreateOrthographicOffCenter(0, 960, 540, 0, -100, 10);  //what I use on a 1920x1080 screen
    Matrix view = Matrix.CreateScale(1); 
    debug.RenderDebugData(ref i, ref view);
}
```

**Comment by nkast** [comment]
The nuget package contains the DiagnosticsFont.xnb and it should include it in the final build.  
The code assumes that ContentManager.Content is set to "Content" folder. 

I've test it when I first created the nuget package but haven't test it since nor I have tested it with the new PackageReference.
https://github.com/tainicom/Aether.Physics2D/blob/56811905f1b8a2182070b95073048285c3c0a8e9/Physics2D.Diagnostics/Package.nuspec#L31
